analyzer-comments/elm/common/no_unused/parameters.md

```diff
@@ -0,0 +1,13 @@
+# parameters 
+
+[comment]: # (This error is raised when a parameter is unused)
+[comment]: # (requires 1 param:)
+[comment]: # (- definition: code extract with unused value)
+
+The parameter indicated below is not used anywhere.
+You could remove it or use it somewhere if it can be used.
```

Comment:
The copy here is different to the previous examples ...

Reply:
Yes, that's on purpose.

For cases like 
```elm
foo =
    let
        ( left, right ) =
            tupleValue
    in
    left
```
Then `right` could be used somewhere, but here
```elm
foo =
    let
        ( _, _ ) =
            bar
    in
    1
```
the pattern `(_, _)` cannot actually be used anywhere.

That's why I say "or use it somewhere if it can be used" in this case. For other rules, like `NoUnused.CustomTypeConstructorArgs`, it's different, whatever argument it catches could always potentially be used.

Reply:
Cool, could you mention that somewhere, in the comment or something?